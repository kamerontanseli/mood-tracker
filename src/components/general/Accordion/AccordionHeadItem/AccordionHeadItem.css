.AccordionHeadItem {
  border-right: 1px solid #eee;
  padding: 0px 10px;
  flex-grow: 1;
  text-align: center;
}

.AccordionHeadItem:first-child {
  padding-left: 0px;
}

.AccordionHeadItem:last-child {
  border-right: none;
  padding-right: 0px;
}
