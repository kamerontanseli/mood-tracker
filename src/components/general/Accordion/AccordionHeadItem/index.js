import "./AccordionHeadItem.css";
import React from "react";

const AccordionHeadItem = ({ children }) => (
  <div className="AccordionHeadItem">{children}</div>
);

export default AccordionHeadItem;
