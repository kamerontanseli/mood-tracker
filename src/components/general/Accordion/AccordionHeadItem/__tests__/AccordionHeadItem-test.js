import React from 'react'
import renderer from 'react-test-renderer'
import AccordionHeadItem from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <AccordionHeadItem>Hello world</AccordionHeadItem>
    );
    expect(snapshot).toMatchSnapshot()
  })
})