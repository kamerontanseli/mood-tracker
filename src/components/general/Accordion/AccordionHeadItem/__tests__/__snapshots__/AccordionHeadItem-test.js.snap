// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<div
  className="AccordionHeadItem"
>
  Hello world
</div>
`;
